cmd/subnetcmd/upgradecmd/export_upgrade.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package upgradecmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/utils/storage"
+	"github.com/spf13/cobra"
+)
+
+// avalanche subnet upgrade import
+func newUpgradeExportCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export [subnetName]",
+		Short: "Export the upgrade bytes file to a location of choice on disk",
+		Long:  `Export the upgrade bytes file to a location of choice on disk`,
+		RunE:  upgradeExportCmd,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	cmd.Flags().StringVar(&upgradeBytesFilePath, upgradeBytesFilePathKey, "", "Export upgrade bytes file to location of choice on disk")
+	if err := cmd.MarkFlagRequired(upgradeBytesFilePathKey); err != nil {
+		panic(err)
+	}
+
+	return cmd
+}
+
+func upgradeExportCmd(cmd *cobra.Command, args []string) error {
+	subnetName := args[0]
+	if !app.GenesisExists(subnetName) {
+		ux.Logger.PrintToUser("The provided subnet name %q does not exist", subnetName)
+		return nil
+	}
+
+	if _, err := os.Stat(upgradeBytesFilePath); err == nil {
+		ux.Logger.PrintToUser("The file specified with path %q already exists!", upgradeBytesFilePath)
+
+		yes, err := app.Prompt.CaptureYesNo("Should we overwrite it?")
+		if err != nil {
+			return err
+		}
+		if !yes {
+			ux.Logger.PrintToUser("Aborted by user. Nothing has been exported")
+			return nil
+		}
+	}
+
+	subnetPath := filepath.Join(app.GetUpgradeFilesDir(), subnetName)
```

Comment:
Should we push this whole path generation function into `app`?

Reply:
I will reuse the `pkg/subnet/upgrades` package for this